style.css

```diff
@@ -297,3 +301,116 @@
   color: #7a869a;
   rotate: 90deg;
 }
+
+.contact-h {
+  font-family: " Poppins", sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-p {
+  font-family: " Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #ebebff;
+  width: 100%;
+}
+
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(image-folder/Contact1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+  width: 315px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+  display: flex;
+  flex-direction: column;
+  margin-left: 330px;
+}
```

Comment:
- [ ] Kindly make sure the contact section is aligned well on the page in order to match the provided Figma design.

- [OPTIONAL] Kindly make sure to align the rest part of your Portfolio correctly in order to match the provided Figma design.

 - ### YOUR
    ![screencapture-127-0-0-1-5500-index-html-2023-04-06-12_41_32](https://user-images.githubusercontent.com/87197412/230354165-80494cb4-25a7-4a98-a1f0-37de8bf53467.png)
 DESIGN
 
 - ### FIGMA DESIGN
![image](https://user-images.githubusercontent.com/87197412/230354844-e5c5c066-67ba-4637-900a-d5e5b410bf64.png)